force-app/main/default/classes/StaffAvailabilityController.cls

```diff
@@ -106,8 +106,12 @@ public with sharing class StaffAvailabilityController {
 			} else {
 				contact[0].Status__c = Constants.NOT_AVAILABLE_STATUS;
 			}
-			SecureDml updateObject = new SecureDml();
-			updateObject.dmlUpdate(contact);
+			/*SecureDml updateObject = new SecureDml();
+			updateObject.dmlUpdate(contact);*/
+			Database.SaveResult saveResult = Database.update(contact[0], false);
```

Comment:
SecureDML checks for CRUD, why remove?

Reply:
@psharma-traction  please see note above, secureDML is recommended.

Reply:
updated with the SecureDML.